pkg/sdk/testint/external_volumes_gen_integration_test.go

```diff
@@ -0,0 +1,605 @@
+			*sdk.NewExternalVolumeStorageLocationRequest().WithS3StorageLocationParams(
+				*sdk.NewS3StorageLocationParamsRequest(
+					s3StorageLocations[0].S3StorageLocationParams.Name,
+					s3StorageLocations[0].S3StorageLocationParams.StorageProvider,
+					s3StorageLocations[0].S3StorageLocationParams.StorageAwsRoleArn,
+					s3StorageLocations[0].S3StorageLocationParams.StorageBaseUrl,
+					*sdk.NewExternalVolumeS3EncryptionRequest(
+						s3StorageLocations[0].S3StorageLocationParams.Encryption.Type,
+					).WithKmsKeyId(*s3StorageLocations[0].S3StorageLocationParams.Encryption.KmsKeyId),
+				).WithStorageAwsExternalId(*s3StorageLocations[0].S3StorageLocationParams.StorageAwsExternalId),
+			),
+		)
+
+		err := client.ExternalVolumes.Alter(ctx, req)
+		require.NoError(t, err)
+
+		props, err := client.ExternalVolumes.Describe(ctx, id)
+		require.NoError(t, err)
+
+		trimmedProperties := trimProperties(t, props)
+		assert.Equal(t, 5, len(trimmedProperties))
+		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "COMMENT", Value: comment})
+		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "ALLOW_WRITES", Value: strconv.FormatBool(allowWrites)})
+		assert.Contains(t, trimmedProperties, ExternalVolumePropNameValue{Name: "ACTIVE", Value: ""})
+		assert.Contains(
+			t,
+			trimmedProperties,
+			ExternalVolumePropNameValue{
+				Name: "STORAGE_LOCATION_1",
+				Value: fmt.Sprintf(
+					`{"NAME":"%s","STORAGE_PROVIDER":"GCS","STORAGE_BASE_URL":"%s","ENCRYPTION_TYPE":"%s"}`,
+					gcsStorageLocationsNoEncryption[0].GCSStorageLocationParams.Name,
+					gcsStorageLocationsNoEncryption[0].GCSStorageLocationParams.StorageBaseUrl,
+					gcsStorageLocationsNoEncryption[0].GCSStorageLocationParams.Encryption.Type,
+				),
+			},
+		)
+		assert.Contains(
+			t,
+			trimmedProperties,
+			ExternalVolumePropNameValue{
+				Name: "STORAGE_LOCATION_2",
+				Value: fmt.Sprintf(
+					`{"NAME":"%s","STORAGE_PROVIDER":"%s","STORAGE_BASE_URL":"%s","STORAGE_AWS_ROLE_ARN":"%s","STORAGE_AWS_EXTERNAL_ID":"%s","ENCRYPTION_TYPE":"%s","ENCRYPTION_KMS_KEY_ID":"%s"}`,
+					s3StorageLocations[0].S3StorageLocationParams.Name,
+					s3StorageLocations[0].S3StorageLocationParams.StorageProvider,
+					s3StorageLocations[0].S3StorageLocationParams.StorageBaseUrl,
+					s3StorageLocations[0].S3StorageLocationParams.StorageAwsRoleArn,
+					*s3StorageLocations[0].S3StorageLocationParams.StorageAwsExternalId,
+					s3StorageLocations[0].S3StorageLocationParams.Encryption.Type,
+					*s3StorageLocations[0].S3StorageLocationParams.Encryption.KmsKeyId,
+				),
+			},
+		)
+	})
+
+	t.Run("Describe", func(t *testing.T) {
+		allowWrites := true
+		comment := "some comment"
+		id := createExternalVolume(
+			t,
+			append(append(append(append(s3StorageLocations, gcsStorageLocationsNoEncryption...), azureStorageLocations...), s3StorageLocationsNoEncryption...), gcsStorageLocations...),
```

Comment:
Once [version 1.22](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/go.mod#L3) is available [Slices.Concat](https://pkg.go.dev/slices#Concat) can be used, which will clean these lines up a bit